docker-compose/embeddings/embeddings.docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
   #
   embeddings:
     container_name: embeddings
-    image: 'index.docker.io/sourcegraph/embeddings:insiders@sha256:0e4f28307fb942d57a7ef0963e53e9fcf99c1d34dfaf2ba5345d9faecb5954f1'
+    image: 'index.docker.io/sourcegraph/embeddings:insiders@sha256:6bfffb3f0d15a4f71d70989656a35d617021574b9081304602e310436ce9f3f9'
```

Comment:
cc @stefanhengl also using insiders tag here, but atleast in this repo executors uses a tagged version.

Reply:
hah, the CI over here fails due to the insiders tag. I wonder why previous releases worked. the CI must of become more strict. I am gonna manually add a tag and test that it exists and has the same hash.

Reply:
Well this is a good catch, I'll open a PR for the othe rrepo which is also using insiders. But this sha256 is not for the released tag.